ocaml/xapi-storage/generator/lib/data.ml

```diff
@@ -132,6 +140,23 @@ module Datapath (R : RPC) = struct
       ]
       (dbg @-> uri_p @-> domain @-> returning unit error)
 
+  let import_activate =
```

Comment:
so this new method would only be availab for SMAPIv3 SRs? The migration for SMAPIv1 SRs would still use the old way where xapi is responsible for constructing the path?